MediaGallerySynchronization/composer.json

```diff
@@ -5,6 +5,7 @@
         "php": "~7.3.0||~7.4.0",
         "magento/framework": "*",
         "magento/module-media-gallery-api": "*",
+        "magento/module-media-gallery-renditions-api": "*",
```

Comment:
The dependency can be reveresed by extending configuration from renditions module